protocol.html

```diff
@@ -369,6 +369,9 @@ <h3 property="schema:name skos:prefLabel">Terminology</h3>
                     <dt about="#root-container" property="skos:prefLabel" typeof="skos:Concept"><dfn id="root-container">root container</dfn></dt>
                     <dd about="#root-container" property="skos:definition">A root container is a container resource that is at the highest level of the collection hierarchy.</dd>
 
+                    <dt about="#authoritative-information" property="skos:prefLabel" typeof="skos:Concept"><dfn id="authoritative-information">authoritative information</dfn></dt>
+                    <dd about="#authoritative-information" property="skos:definition">Authoritative information encompasses data about resources described by means of RDF statements [<cite><a class="bibref" href="#bib-rdf11-concepts">RDF11-CONCEPTS</a></cite>] supplied by clients, servers, and intermediaries</dd>
```

Comment:
I'm not sure I can find counterexamples here; to me it seems this is _all_ RDF data. Is that correct? Why is it authoritative?

Reply:
I've updated the comment to include an overview which was originally intended as a response to your question here. Hope that helps.

Reply:
I think the definition needs to be more precise. For it to have value as a definition, it needs to concisely describe what is meant in this context. In this context, we are talking about server-supplied metadata, right?

Reply:
? No. As said, it is a broader concept covering the sender's intention. What can be improved in the definition is what constitutes authoritative. I've expressed that elsewhere with references, but will look into revising.

Reply:
Yes, but the broad concept doesn't seem useful to us, AFAICT. Anyway, looking forward to clarifications.

Reply:
I've expressed it around the concept of "resource metadata" in place of "authoritative information" in https://github.com/solid/specification/pull/352/commits/1727bb80ac6e8bf026d69d16045ed217b11b6cf8 . We can come back to "authoritative information" later e.g., after "server description" (Issue #355 ).